hooks/submitter_create.py

```diff
@@ -40,7 +41,10 @@ def can_submit(self):
         """
 
         if not self.__create_client_module.is_create_installed():
-            self.log_warning("Shotgun Create is not installed.")
+            QtGui.QMessageBox.warning(
+                None, "Cannot submit to Shotgun", "Shotgun Create is not installed!",
```

Comment:
Could we provide a hyperlink to where to download in the box?

Reply:
The design is to open the browser. The next call open your browser on the right page.

Reply:
```
self.__create_client_module.open_shotgun_create_download_page(
                 self.__app.sgtk.shotgun		                 self.__app.sgtk.shotgun
             )
)
```

Reply:
Uh, what, how did I miss this! Sorry!